akka-docs/src/main/paradox/stream/operators/ActorSource/actorRefWithBackpressure.md

```diff
@@ -34,7 +34,9 @@ See also:
 
 ## Example
 
-With `actorRefWithBackpressure` two actors get into play: one which feeds the stream and is materialized when the stream runs, and one which is provided by us and gets the ack signal when an element is emitted into the stream.
+With `actorRefWithBackpressure` two actors get into play: 
+1. An actor that is materialized when the stream runs. It feeds the stream.
```

Comment:
I think there must be an empty line before the list

Reply:
Done